### ISSUE TITLE
Dynamis Beaucedine Attestation drop from Quiebetiel

### ISSUE THREAD

**Issue by Raphtalia1** [EdenServer/community#3380]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details
Quiebitiel in Dynamis-Beaucedine did not drop an attestation, all attestation NM's should drop 1/3 of their possible attestations on every kill, with a small chance for 100 piece or in the case of Gaublefaupe the Shield Attestation
![Screenshot 2020-10-28 19 26 53](https://user-images.githubusercontent.com/50685071/97516250-55aab780-1958-11eb-9926-28cbe788bec2.png)

**Comment by Shoruto** [comment]
2 posts to support this report that it should drop at least 1 and 2 to claim chance of no attestation drops.  https://www.bluegartr.com/threads/39229-Attestation-Drop-Rates

Finding no other era information about drop rate only ooe information.

 **2006-07-30**
tumu

Attestation Drop Rates

   We've killed the NM **5 times** out of 5 tries, but so far we are 0/5 on the right attestation. I think we've got **Attestation of Might 3/5 and Attestation of Accuracy 2/5.**

---------------------------------------------------------------------------------------------------------

 **2006-07-30** 21:52
SwampdonkeyPLD

  A guy in my linkshell is **0/10** on club. We have gotten **9 daggers and 1 horn**. 

---------------------------------------------------------------------------------------------------------

1 post to confirm that there is a slight chance no Attestation drops based on a screenshot on KI (can't find)

 **2006-07-30** 22:27
Lordwafik

  **All the attestation mobs have a chance to drop nothing at all, there was a SS on KI, same SS had him dropping an M.silver instead of attestation.** 

----------------------------------------------------------------------------------------------------------

1 post to support the above post by Lordwafik

 **2006-07-30**
Wizerd

   Quote Originally Posted by Lordwafik
   All the attestation mobs have a chance to drop nothing at all, there was a SS on KI, same SS had him dropping an M.silver instead of attestation.
    **Like, 1% or something?o.O**
